starter-code/src/functions-and-arrays.js

```diff
@@ -1,16 +1,64 @@
 // Iteration #1: Find the maximum
 
+function maxOfTwoNumbers(num1, num2) {
+  if(num1 > num2) {
+    return num1
+  } else {
+    return num2
+  }
```

Comment:
You can also do it with the ternary operator shorthand notation like return num1 > num2 ? num1 : num2